.github/workflows/test-pygenesys.yml

```diff
@@ -0,0 +1,66 @@
+# Preamble
+name: Test Pygenesys
+
+on:
+  push:
+    paths:
+      - 'pygenesys/**'
+      - 'pygenesys/tests/**'
+  pull_request:
```

Comment:
I think this will download Temoa to `pygenesys\pygenesys\temoa` (per the log file), which is not really where temoa should live in relation to pygenesys.

Reply:
The block you've highlighted doesn't really effect where Temoa is downloaded to. Instead, it indicates that changes to files on the given paths will trigger this workflow.